ducktape/command_line/main.py

```diff
@@ -171,6 +171,7 @@ def main():
     # Run the tests
     runner = SerialTestRunner(session_context, tests)
     test_results = runner.run_all_tests()
+    session_context.close_logger()
```

Comment:
Would it make sense to just have a `close()` which, if necessary, can handle cleaning up other resources too?